daml-lf/language/daml-lf.bzl

```diff
@@ -59,9 +59,12 @@ PROTO_LF_VERSIONS = LF_VERSIONS
 # The subset of LF versions accepted by //daml-lf/encoder
 ENCODER_LF_VERSIONS = ["1.dev" if ver == "dev" else ver for ver in LF_VERSIONS]
 
+# We support older LF versions using an older compiler binary
+LEGACY_COMPILER_LF_VERSIONS = ENCODER_LF_VERSIONS
+
 # The subset of LF versions accepted by the compiler in the syntax
 # expected by the --target option.
-COMPILER_LF_VERSIONS = ENCODER_LF_VERSIONS
+COMPILER_LF_VERSIONS = [ver for ver in ENCODER_LF_VERSIONS if ver not in ["1.6", "1.7", "1.8", "1.11", "1.12", "1.13"]]
```

Comment:
The list `["1.6", "1.7", "1.8", "1.11", "1.12", "1.13"]` is what I expected `LEGACY_COMPILER_LF_VERSIONS` to be.

If so you could rewrite this as `[ver for ver in ENCODER_LF_VERSIONS if ver not in LEGACY_COMPILER_LF_VERSIONS]`.

Reply:
changed it to that and added `all_compiler_lf_versions` corresponding to the previous behavior

Reply:
Thanks!